package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "hexo-filter-nofollow",
+  "version": "2.0.0",
+  "description": "Add nofollow attribute to all external links automatically",
+  "main": "index.js",
+  "directories": {
+    "lib": "./lib"
+  },
+  "files": [
+    "lib/",
+    "index.js"
+  ],
+  "engines": {
+    "node": ">= 8.6.0"
+  },
+  "scripts": {
+    "eslint": "eslint .",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
IMHO you can choose anything test framework you want when you write a unit test code.

Reply:
Or even a dummy test like https://github.com/hexojs/hexo-uglify, but it can be done in a separate PR.

Reply:
A unit test might be added later in another PR.